scipy/sparse/csgraph/_laplacian.py

```diff
@@ -112,6 +112,9 @@ def laplacian(
     The normalization is symmetric, making the normalized Laplacian also
     symmetric if the input csgraph was symmetric.
 
+    The algorithmic complexity is for a graph with V
+    vertices is ``O(V^2)``.
```

Comment:
Ah sorry I meant we should make all of them monospace not just this one

Reply:
Okay. I thought only this specific location is causing issues. Sure. Give me a few minutes, I will update everywhere and push.

Reply:
Addressed.